cli/exchange/version.go

```diff
@@ -3,17 +3,31 @@ package exchange
 import (
 	"fmt"
 	"github.com/open-horizon/anax/cli/cliutils"
+	"strings"
 )
 
-func Version(org, userPw string) {
-	cliutils.SetWhetherUsingApiKey(userPw)
-	credToUse := ""
+func Version(org string, userPw string) {
+	output := LoadExchangeVersion(true, org, userPw)
+	fmt.Println(output)
+}
+
+// Loads exchange version based on optional org and optional set of prioritized credentials (first non-empty credentials will be used)
+// If loadWithoutCredentials is false and credentials are missing - the version loading will not be executed (empty string will be returned)
+func LoadExchangeVersion(loadWithoutCredentials bool, org string, userPws ...string) string {
+	var credToUse string
+	for _, cred := range userPws {
+		if cred != "" {
+			credToUse = cred
+		}
+	}
+
+	cliutils.SetWhetherUsingApiKey(credToUse)
```

Comment:
credToUse could be empty. Do we need this?

Reply:
Moved to non-empty creds section